contents/blog/announcing-notebooks.md

```diff
@@ -0,0 +1,43 @@
+---
+date: 2023-12-17
+title: "Announcing Notebooks for PostHog"
```

Comment:
```suggestion
title: "Introducing Notebooks for PostHog"
```

Reply:
I prefer this to announcing tbh, but don't feel strongly.

Reply:
Agreed - can we change the slug here too?